src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -39,47 +36,48 @@ public void setUp() {
     Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/activityfeed.jsp"))
         .thenReturn(mockRequestDispatcher);
 
-    mockUserStore = Mockito.mock(UserStore.class);
-    activityFeedServlet.setUserStore(mockUserStore);
-
-    mockConversationStore = Mockito.mock(ConversationStore.class);
-    activityFeedServlet.setConversationStore(mockConversationStore);
-
-    mockMessageStore = Mockito.mock(MessageStore.class);
-    activityFeedServlet.setMessageStore(mockMessageStore);
+    mockEventStore = Mockito.mock(EventStore.class);
+    activityFeedServlet.setEventStore(mockEventStore);
   }
 
   @Test
   public void testDoGet() throws IOException, ServletException {
+    List<Event> fakeEventList = new ArrayList<>();
+
     UUID fakeUserId = UUID.randomUUID();
     User fakeUser = 
```

Comment:
Not really sure if you need a fake user since you can just manually fill in dummy values to create the event object.